cdap-ui/app/cdap/components/CaskWizards/StreamCreateWithUpload/index.js

```diff
@@ -71,6 +71,11 @@ export default class StreamCreateWithUploadWizard extends Component {
             let filename = state.upload.filename;
             let filetype = 'text/' + filename.split('.').pop();
             let authToken = cookie.load('CDAP_Auth_Token');
+            if (typeof fileContents !== 'object') {
+              return Rx.Observable.create((observer) => {
+                observer.onNext();
```

Comment:
TL;DR- User with only create permission but not ingest will see issues in wizard.

Its technically not an error. We just skip that step if no file is uploaded. This surfaced when we tested our new security model. As a user I might have permission to create the stream but not upload. Existing wizard will mandatorily create the stream and do an empty upload. This causes the stream to be created but the wizard will show an error because upload failed.